openbb_terminal/stocks/fundamental_analysis/seeking_alpha_view.py

```diff
@@ -68,6 +68,7 @@ def display_rev_estimates(
             show_index=False,
             title=f"{symbol.upper()} Revenue History and Estimations",
             export=bool(export),
+            columns_keep_types=["fiscalyear"],
```

Comment:
Thanks for jumping in!

A slight oversight on my part, we also need to convert the column to a string. One more line to add above.

Line 66:
```
rev_estimates["fiscalyear"] = rev_estimates["fiscalyear"].astype(str)
```

While we are here, let's also apply the same solution to the other function, `display_eps_estimates`.

Reply:
I think this is actually because you are not running this from the environment that is not where OpenBB is installed.  With the environment active, you would only need to run `pytest {path} {--params}`.

It seems to test fine over here, and oddly enough, the cassettes already captured reflect EPS and Rev as you changed them to be. :)

The linters might yell at us for not having an empty line at the end of the test file, but let's find out!

Thanks!

Reply:
ok, I thought that if I build from source and then setup the env using `poetry env use python3.10` then things should've worked as mentioned. I will recheck on my side.


You're welcome and thanks again for the description, Indeed exicted to start my contribution streak to OpenBB and in general to contribute to the openBB ecosystem.